mod_reforged/hooks/skills/backgrounds/character_background.nut

```diff
@@ -1,9 +1,17 @@
 ::Reforged.HooksMod.hook("scripts/skills/backgrounds/character_background", function(q) {
+	q.m.CalculatedDailyCost <- 0;
```

Comment:
I can't see where this `CalculatedDailyCost` variable actually used e.g. to set the daily wage.